internal/expr/prepare.go

```diff
@@ -9,7 +9,7 @@ import (
 	"strings"
 )
 
-// PreparedExpr contains an SQL expression that is ready for execution.
+// PreparedExpr represents a SQL statement for use in a query.
```

Comment:
The original comment, while not a great comment indeed, was hinting at the expression pipeline composed by ParsedExpr, PreparedExpr, QueryExpr. The new comment loses that, and doesn't really improve the understanding much (all of those are SQL statements for use in a query, somehow).  Can we have a look at those three and try to improve things a bit further?

Reply:
I've changed all three. This is how they currently look:
```
// ParsedExpr represents a parsed SQL string containing SQLair input and output
// expressions.
```
```
// PreparedExpr is a ParsedExpr that has been validated and processed into SQL
// ready for use in a query.
```
```
// QueryExpr represents a SQL statement and its arguments. It is is ready for
// execution on a database and then the scanning of the query results.
```
The other two comments are in PR #90 and PR #93 respectively.